keps/sig-apps/20190318-PodDisruptionBudget-graduation-to-stable.md

```diff
@@ -107,6 +107,27 @@ The current behavior of the disruption controller for the different types of inp
 types of pods that might be encountered are documented in: 
 https://docs.google.com/spreadsheets/d/12HUundBS-slA6axfQYZPRCeIu_Au_wsGD0Vu_oKAnM8/edit?usp=sharing
 
+To fix this, we will make two adjustments:
```

Comment:
You need to update the KEP to the new format and fill in the Production Readiness section (see the new template in:
https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template#production-readiness-review-questionnaire

Reply:
Yeah, I will take a look at the new format.